### DISCUSSION TITLE
Future of Steam Compatibility ?

### DISCUSSION THREAD

**Discussion by Totor954** [dfint/.github#3] [Q&A] [answered]
Hi, 'm french (Sorry for my bad english for advance)

I just want to ask if it has a project to make the gui updater compatibility wch the steam version ?

**Comment by insolor** [comment]
What gui updater do you mean?

**Reply by Totor954** [reply]
The gui updater to translate dwarf forteresse

**Reply by insolor** [reply]
@Totor954 do you mean some existing gui (maybe https://github.com/dfint/df-translation-client?) or creation of a new gui?

**Reply by Totor954** [reply]
Yes i speak to this application, 

I try to use but i don't success

I have try to use dfrus but he sait : not compatible wich 64 i try to use dlfrus64 but not success too

I'm not an expert of python (and of english :-))

I just want to know one thing : 

i see on transifex translation french is 84%

can we use this tranlation today or no tool exist for the moment ???

Very very sorry for my bad english, i try to make simple sentence

**Reply by insolor** [reply]
@Totor954 there are a lot to update. Now I'm going to update it to download translation files from the dwarf-fortress-steam project separately. Then I'll add the possibility to generate csv file to a new patch, which I'm going to release soon too.

**Reply by Totor954** [reply]
Ok thx for your ansmwer

**Reply by insolor** [reply, marked as answer]
As of today, development of [df-translation-client](https://github.com/dfint/df-translation-client) is frozen, because it's too time consuming to develop and maintain a GUI application.

We created an [autobuild](https://github.com/dfint/autobuild) repository, which contains regulary (once a day) updated csv files which can be used with [df-steam-hook](https://github.com/dfint/df-steam-hook).

In the future we'll be more concentrated on the development of command line tools and automation, for example, to provide ready to use packages for each language.

UPD: as of today, there is a [GUI application](https://github.com/dfint/installer) to install a translation to the game.

**Reply by Totor954** [reply]
Thx for your answer 

just could tou do anything to convert : 
All 
éèêë to e
àâä to a 
… because for each language without font modification That will bug text in game
Me I do This witch « search and replace » of notepad++
but for begginer it’s less easy 

I thons wich your power in python, you can make a parser to transform in no accentues character 

and vert sorry for your eyes for my very bas gramatical english 

so thank you very Much for your personnal investissement

**Reply by insolor** [reply]
Just modify the font (curses_640x300.png) according to latin-9 encoding (https://en.wikipedia.org/wiki/ISO/IEC_8859-15#Codepage_layout). I don't see any reason, why you need to break the text instead of fixing the font.
